src/codeActionProvider.ts

```diff
@@ -55,6 +56,9 @@ async function revealWorkspaceEdit(workspaceEdit: WorkspaceEdit): Promise<void>
 }
 
 export async function lombokAction(params: CodeActionParams, annotations: string[]): Promise<void> {
+    sendInfo("", {
```

Comment:
As for start event, how about instrument the command directly?

Reply:
yes, we can do there.